checker/run

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -uex
+
+. "$(dirname "$0")"/../options
+
+if [ -z "$pattern_extractor" ] | [ -z "$message" ]; then
+    echo "Required fields pattern_extractor and message not found in ../options"
+    exit 1
+fi
+
+new_value=$(eval "$pattern_extractor" | tr -d '\r')
+if [ -z "$new_value" ]; then
+    echo "$pattern_extractor didn't found anything"
```

Comment:
"pattern extractor $pattern_extractor could not extract anyhting"

.. or similar.